conftest.py

```diff
@@ -5,7 +5,7 @@
 from hypothesis import HealthCheck, settings
 
 settings.register_profile(
-    "ci", max_examples=1000, deadline=None, suppress_health_check=[HealthCheck.too_slow]
+    "ci", max_examples=250, deadline=None, suppress_health_check=[HealthCheck.too_slow]
```

Comment:
What are the reasons for setting `deadline` to `None` and suppressing `too_slow` health-checks?
Perhaps we should set the deadline and not suppress to easier catch "bad" tests?

Reply:
The reason for dropping deadline and supressing too_slow on CI is that the CI timing is not predictable. Every once in a while, a test will fail to meet the deadline because the runner had a hiccup. These checks are still in place locally.